prql-compiler/tests/integration/queries/invoice_totals.prql

```diff
@@ -26,7 +25,8 @@ select [
     day,
     num_orders = s"CAST({num_orders} AS INT)",
     num_tracks = s"CAST({num_tracks} AS INT)",
-    total_price = s"ROUND(CAST({total_price} AS FLOAT) * 100) / 100",
+    total_price = s"ROUND(CAST({total_price} AS FLOAT) * 100, 0) / 100",
     running_total_num_tracks = s"CAST({running_total_num_tracks} AS INT)",
     num_tracks_last_week = s"CAST({num_tracks_last_week} AS INT)"
 ]
+take 20
```

Comment:
Great idea to reduce the snapshots.

In another PR we could try adding more prql-specific features here, like `take 10..20`, or using `as` rather than the s-string `CAST`...

Reply:
I see we have another example with the more advanced `take` syntax, and I put in https://github.com/PRQL/prql/pull/2462 for the s-strings